docs/getting-started.rst

```diff
@@ -0,0 +1,249 @@
+       Type                              Name                                 Plan
+   +   pulumi:pulumi:Stack               myproject-mystack                    create
+   +   ├─ tb:network:MultiCidrVpc        myproject-mystack-vpc                create
+   +   │  ├─ aws:ec2:Vpc                 myproject-mystack-vpc                create
+   +   │  ├─ aws:ec2:Subnet              myproject-mystack-vpc-subnet-0       create
+   +   │  ├─ aws:ec2:Subnet              myproject-mystack-vpc-subnet-1       create
+   +   │  └─ aws:ec2:Subnet              myproject-mystack-vpc-subnet-2       create
+   +   ├─ aws:ec2:RouteTableAssociation  myproject-mystack-vpc-subnetassoc-0  create
+   +   ├─ aws:ec2:RouteTableAssociation  myproject-mystack-vpc-subnetassoc-1  create
+   +   └─ aws:ec2:RouteTableAssociation  myproject-mystack-vpc-subnetassoc-2  create
+
+  Resources:
+      + 9 to create
+
+
+Manual Setup
+------------
+
+S3 bucket
+^^^^^^^^^
+
+Create an S3 bucket in which to store state for the project. You must have one bucket devoted to your project, but you
+can store multiple stacks' state files in that one bucket. The bucket should not be public (treat these files as
+sensitive), and it's a good idea to turn on versioning, as it can save you from some difficult situations down the road.
+
+The name of an S3 bucket is used as part of a global domain, and so your bucket name must be globally unique. A good way
+to handle this is to include an organization name in your bucket name. As a template, you may use:
+::
+
+  $ORG-$PROJECT_NAME-pulumi
+
+Repo setup
+^^^^^^^^^^
+
+You probably already have a code repo with your application code in it. If not, create such a repo.
+
+Create a directory there called ``pulumi`` and create a new project and stack in it. You'll need the name of the S3
+bucket from the previous step here. If you are operating in an AWS region other than what is set as your default for
+AWSCLI, be sure to ``export AWS_REGION=us-east-1`` or whatever else you may need to do to override that.
+::
+
+  cd /path/to/pulumi/code
+  pulumi login s3://s3-bucket-name
+  pulumi new aws-python
+
+Follow the prompts to get everything named.
+
+Set up this module
+^^^^^^^^^^^^^^^^^^
+
+Ensure your pulumi code directory contains a ``requirements.txt`` file with at least this repo listed:
+::
+
+  git+https://github.com/thunderbird/pulumi.git
+
+You can pin your code to a specific version of this module by appending ``@branch_name`` to that. For example:
+::
+
+  git+https://github.com/thunderbird/pulumi.git@v0.0.2
+
+Pulumi will need these requirements installed. On your first run of a ``pulumi preview`` command (or some others),
+Pulumi will attempt to set up its working environment. If this fails, or you need to make adjustments later, you can
+activate Pulumi's virtual environment to perform pip changes. Assuming Pulumi's virtual environment lives at ``venv``,
+run:
+::
+
+  ./venv/bin/pip install -U -r requirements.txt
```

Comment:
I would suggest the reader already activate their venv by doing `source ./venv/bin/activate`.

Reply:
Just updated this. Will push the commit soon.